src/TimebarWrapper.tsx

```diff
@@ -56,27 +58,30 @@ const TimebarWrapper = (props: any) => {
           setTimerange(start, end)
         }}
       >
-        {(props: any) =>
-          loading ? (
+        {(props: any) => {
+          return loading ? (
             <Loader />
           ) : (
-            <>
+            <Fragment>
```

Comment:
Isn't `<>` equivalent to `<Fragment>` ?

Reply:
Yes, it is. I changed because it is clearer for me to read but I can use `</>` if we agree as standard